pxr/imaging/hdMtlx/hdMtlx.cpp

```diff
@@ -585,6 +591,23 @@ HdMtlxCreateMtlxDocumentFromHdMaterialNetworkInterface(
         mxType,
         mxShaderNode);
 
+    // just for testing...
+    bool localDebug = false;
```

Comment:
There's a system in hydra called debug codes that would be perfect here.  The idea is that you create a local "debugCodes.h/.cpp", like the ones in imaging/hd (for example); add a new debug code, such as HDMTLX_VERSION_UPGRADE; and here, replace "localDebug" with TfDebug::IsEnabled(HDMTLX_VERSION_UPGRADE).

You can trigger this either with the debug codes editor in usdview, or by setting the environment variable "TF_DEBUG" to "HDMTLX_VERSION_UPGRADE".

Having this as a debug code seems like a fantastic idea!